docs/how-to/deploy-sepolia-l1.md

```diff
@@ -0,0 +1,46 @@
+# Deploy Kurtosis CDK with Sepolia as L1
+
+Deploying to Sepolia requires changing the L1-related parameters.
+
+## params.yml
+Change `deploy_l1` to `false`.
+```
+# Deploy local L1.
+deploy_l1: false
+```
+
+Change the `l1_chain_id`, `l1_preallocated_mnemonic`, `l1_rpc_url`, `l1_ws_url`, and `l1_funding_amount` as needed.
+```
+l1_chain_id: 11155111
+# The first account for the mnemonic should have enough funds to cover the deployment costs and the l1_funding_amount transfers to L1 addresses.
+l1_preallocated_mnemonic: <mnemonic_for_a_funded_L1_account>
+# The amount of initial funding to L1 addresses performing services like the Sequencer, Aggregator, Admin, Agglayer, Claimtxmanager. 
+l1_funding_amount: 1ether
```

Comment:
I think we could even skip the deployment of contracts by specifying the addresses of the different contracts already deployed on Sepolia maybe?